test/interpreter/function-stdevpa.spec.ts

```diff
@@ -0,0 +1,22 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function STDEVPA', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=STDEVPA()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate standard deviation (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=STDEVPA(2, 3)'],
+      ['=STDEVPA(B2:D2, E2, F2)', 2, 3, 'foo', 'bar', 4],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.5)
+    expect(engine.getCellValue(adr('A2'))).toBeCloseTo(1.6, 6)
+  })
+})
```

Comment:
Please add missing tests and make it obvious that those arguments are supported or not:

- Arguments can be the following: numbers; names, arrays, or references that contain numbers; text representations of numbers; or logical values, such as TRUE and FALSE, in a reference.
- Text representations of numbers that you type directly into the list of arguments are counted.
- Arguments that contain TRUE evaluate as 1; arguments that contain text or FALSE evaluate as 0 (zero).
- If an argument is an array or reference, only values in that array or reference are used. Empty cells and text values in the array or reference are ignored.
- Arguments that are error values or text that cannot be translated into numbers cause errors.
- If you do not want to include logical values and text representations of numbers in a reference as part of the calculation, use the STDEVP function.

We should mark the requirements and the differences between functions clearly 
Source: https://support.microsoft.com/en-us/office/stdevpa-function-5578d4d6-455a-4308-9991-d405afe2c28c

> Unlike the STDEV function, includes values of type Text and Logical. Text values are treated as number 0. Logical TRUE is treated as 1, and FALSE is treated as 0. Empty cells are not included.
>
> In the sequence, only Numbers and Logical types are considered; cells with Text are converted to 0; other types are ignored. If Logical types are a distinct type, they are still included, with TRUE considered 1 and FALSE considered 0. Any Sample may be of type ReferenceList.
>
> The handling of string constants as parameters is implementation-defined. Either, string constants are converted to numbers, if possible and otherwise, they are treated as zero, or string constants are always treated as zero.

https://docs.oasis-open.org/office/OpenDocument/v1.3/csprd02/part4-formula/OpenDocument-v1.3-csprd02-part4-formula.html#__RefHeading__1018772_715980110